backend/src/main/java/com/bakdata/conquery/io/storage/StoreInfo.java

```diff
@@ -49,76 +49,77 @@
  */
 @RequiredArgsConstructor
 @Getter
-@ToString(of={"name", "keyType", "valueType" })
+@ToString(of = {"name", "keyType", "valueType"})
 public enum StoreInfo implements IStoreInfo {
-    DATASET(Dataset.class, Boolean.class),
-    ID_MAPPING(PersistentIdMap.class, Boolean.class),
-    NAMESPACES(DatasetRegistry.class, Boolean.class),
-    SLAVE(ShardNodeInformation.class, Boolean.class),
-    DICTIONARIES(Dictionary.class, DictionaryId.class),
-    IMPORTS(Import.class, ImportId.class),
-    SECONDARY_IDS(SecondaryIdDescription.class, SecondaryIdDescriptionId.class),
-    TABLES(Table.class, TableId.class),
-    CONCEPTS(Concept.class, ConceptId.class),
-    BUCKETS(Bucket.class, BucketId.class),
-    C_BLOCKS(CBlock.class, CBlockId.class),
-    WORKER(WorkerInformation.class, Boolean.class),
-    EXECUTIONS(ManagedExecution.class, ManagedExecutionId.class),
-    AUTH_ROLE(Role.class, RoleId.class),
-    AUTH_USER(User.class, UserId.class),
-    AUTH_GROUP(Group.class, GroupId.class),
-    STRUCTURE(StructureNode[].class, Boolean.class),
-    FORM_CONFIG(FormConfig.class, FormConfigId.class),
-    WORKER_TO_BUCKETS(WorkerToBucketsMap.class, Boolean.class);
+	DATASET(Dataset.class, Boolean.class),
+	ID_MAPPING(EntityIdMap.class, Boolean.class),
+	NAMESPACES(DatasetRegistry.class, Boolean.class),
+	SLAVE(ShardNodeInformation.class, Boolean.class),
```

Comment:
```suggestion
	SHARD(ShardNodeInformation.class, Boolean.class),
```
Das muss ich übersehen haben, kannst du das bitte noch anpassen?

Reply:
die wird nirgends benutzt, hab sie gelöscht